pkg/minikube/bootstrapper/certs.go

```diff
@@ -478,3 +491,44 @@ func canRead(path string) bool {
 	defer f.Close()
 	return true
 }
+
+// isValid checks a cert/key path and makes sure it's still valid
```

Comment:
I think it would be good to add to the comment that the cert/key paths are removed if they're invalid as based on the func name alone I wouldn't expect it to be deleting things

Reply:
good call, fixed.